rust/worker/src/segment/record_segment.rs

```diff
@@ -632,6 +632,10 @@ impl RecordSegmentFlusher {
 
         Ok(flushed_files)
     }
+
+    pub fn total_keys(&self) -> Option<u64> {
```

Comment:
nit: can this be pub(crate) instead of blanket pub?

Reply:
+1